.cloud-build/cleanup/resource_cleanup_manager.py

```diff
@@ -156,3 +157,29 @@ class BatchPredictionJobCleanupManager(VertexAIResourceCleanupManager):
 
 class ExperimentCleanupManager(VertexAIResourceCleanupManager):
     vertex_ai_resource = aiplatform.Experiment
+
+class BucketCleanupManager(VertexAIResourceCleanupManager):
+    vertex_ai_resource = storage.bucket.Bucket
+
+    def list(self) -> Any:
+        storage_client = storage.Client()
+        return [ bucket for bucket in storage_client.list_buckets()]
+
+    def delete(self, resouce):
+        if not 'tests' in resouce.name:
+            return
+        try:
+            resoure.delete(force=True)
+        except:
+            pass
+
+    @property
+    def type_name(self) -> str:
+        return str(type(self.vertex_ai_resource))
+
+    def get_seconds_since_modification(self, resource: Any) -> float:
+        created = resource.time_created
```

Comment:
`time_created` is creation time and not modification time.

Reply:
I looked at the bucket properties and could not find one for last update time

Reply:
Ah I see, I guess this works as well.
Can you add a comment to that effect for future viewers?

Reply:
Also, you initialized `created` but you don't use it.

Lastly, you should be returning 'second since creation', not the timestamp of when it was created. You'll have to take the difference between the current time and the timestamp. See the other examples in the file.